docset/winserver2022-ps/activedirectory/Get-ADPrincipalGroupMembership.md

```diff
@@ -269,7 +287,7 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: None
+Default value: DefaultNC; Provider: Default is to use the Partition that you are currently in. Else, use DefaultNC (IE: If you are in the RootDSE)
```

Comment:
```suggestion
Default value: DefaultNC; Provider: The default is to use the Partition that you are currently in. Otherwise, use DefaultNC (that is, if you are in the RootDSE)
```